packages/pymedphys_dicom/src/pymedphys_dicom/dicom/anonymise.py

```diff
@@ -142,43 +95,43 @@ def anonymise_dataset(
 
     Parameters
     ----------
-    ds : pydicom.dataset.Dataset
+    ds : `pydicom.dataset.Dataset`
```

Comment:
Do we want double ` here on either side?

Reply:
I did wonder that. We have singles all over the place though and I wondered whether italics were sufficient. But we can convert to doubles if you prefer :)

Reply:
Yeah I know. I think the singles are there from accidentally thinking it's markdown...

Reply:
Done (for `anonymise.py` as least :) See https://github.com/pymedphys/pymedphys/pull/437/commits/b51ce6f8dd2720eca9ae62bc6e6b9b3dd9565200